web/src/components/common.js

```diff
@@ -36,6 +36,17 @@ function addS (count, s = 's') {
   }
 }
 
+function prepareLabels (histo) {
```

Comment:
What about fixing this in how elasticsearch returns the date: https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-aggregations-bucket-datehistogram-aggregation.html#_keys_2

Reply:
done